### PR TITLE
Params + Unmapped names

### PR DESCRIPTION
**Pull request by thiakil**
Adds the discussed stuff for unmapped names, and adds params into the srg list so they can be retrieved.

### CODE REVIEW

File: src/main/java/com/tterrag/k9/mcp/SrgDatabase.java [hunk 1]
```diff
@@ -65,6 +94,21 @@ public void reload() throws IOException {
                 String newOwner = mapping.getOwner();
                 if (existing == null || (owner != null && newOwner != null && owner.length() > newOwner.length())) {
                     srgs.put(mapping.getType(), mapping.getSRG(), mapping);
+                    if (mapping.getType() == MappingType.METHOD && mapping.getSRG().startsWith("func_")){
```
Comment (tterrag1098) [inline src/main/java/com/tterrag/k9/mcp/SrgDatabase.java:97]: Could this maybe be abstracted inside the factory?

File: src/main/java/com/tterrag/k9/commands/CommandMCP.java [hunk 2]
```diff
@@ -120,7 +124,32 @@ public void process(CommandContext ctx) throws CommandException {
...
         } else {
-            builder.append("No information found!");
+            List<ISrgMapping> srgMappings = srgs.lookup(type, name);
+            if (!srgMappings.isEmpty()) {
+                for (ISrgMapping srg : srgMappings){
+                    builder.append("\n");
+                    if(builder.length() > 1) {
+                        builder.append("\n");
+                    }
+                    if(type == MappingType.PARAM){
```
Comment (tterrag1098) [inline src/main/java/com/tterrag/k9/commands/CommandMCP.java]: This is a lot of duplicated code, perhaps it could be merged into one method?

**Review by tterrag1098 — CHANGES_REQUESTED** [review]

File: src/main/java/com/tterrag/k9/mcp/SrgDatabase.java (hunk 1 above)
Comment (thiakil) [reply src/main/java/com/tterrag/k9/mcp/SrgDatabase.java:97]: In what sense? It needs access to the static_methods list of the DB

**Comment by thiakil** [comment]
Unmapped entries for testing: `p_189940_7_` & `func_193984_c`

**Comment by tterrag1098** [comment]
I've done my own version of this, along with some pretty heavy refactors of the whole MCP API.

**Pull request closed without merging**